domain/application/resource/fileresourcestore.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package resource
+
+import (
+	"context"
+	"io"
+
+	"github.com/juju/juju/core/objectstore"
+	coreresource "github.com/juju/juju/core/resources"
+	"github.com/juju/juju/internal/charm/resource"
+)
+
+// FileResourceStore is a ResourceStore for file resource types.
+type FileResourceStore struct {
```

Comment:
todo: make the FileResourceStore private to the package, not need for to be exported.

Reply:
Done!